docs/admin/upgrade.md

```diff
@@ -0,0 +1,10 @@
+# Upgrading the App
+
+Here you will find any steps necessary to upgrade the App in your Nautobot environment.
+
+## Upgrade Guide
```

Comment:
As with the Uninstall case: I think we should do a "write once" guide in Core for this.

Any one-offs can be noted here.